bcda/web/api_test.go

```diff
@@ -427,6 +427,11 @@ func bulkEOBRequestNoBeneficiariesInACOHelper(endpoint string, s *APITestSuite)
 	handler.ServeHTTP(s.rr, req)
 
 	assert.Equal(s.T(), http.StatusInternalServerError, s.rr.Code)
+
+	// NOTE: This relies on the fact that this caller was the last
```

Comment:
I don't fully understand this new Note.  Can you elaborate?

Reply:
This might be moot if we end up including the job creation in a transaction, but I don't have any hooks on retrieving our proposed jobID, so I need to search for jobs based on ACOs.

If there's another test running in parallel in this one that happens to be inserting another job for the same ACO, this test may fail because we'll pick up the other job (not the one we had created) and that job may not be failed.

Reply:
Ahhh, makes sense.